lib/src/models/data/home.dart

```diff
@@ -30,4 +33,41 @@ class HomeModel extends Model {
 		notifyListeners();
 		_messageTimer = Timer(const Duration(seconds: 5), () { message = null; notifyListeners(); });
 	} 
+
+  /// Starts a new timer
+  void startTimer({required MissionTimer timer}){
```

Comment:
Really nitpicky, as most of my comments are, but you can just make it unnamed, since it's the only parameter and `startTimer(timer: timer)` is a bit repetitive